src/Microsoft.DotNet.Web.ProjectTemplates/content/StarterWeb-CSharp/wwwroot/css/site.css

```diff
@@ -1,4 +1,5 @@
-﻿/* Sticky footer styles
+﻿/* Please see documentation at https://docs.microsoft.com/aspnet/core/client-side/bundling-and-minification\ 
```

Comment:
This doesn't look right in terms of closing the comment, and then a new comment starts on the next line?